ui/src/router/components/configuration/components/router_config_section/RoutesConfigTable.js

```diff
@@ -59,6 +59,7 @@ export const RoutesConfigTable = ({ routes, rules = [], defaultRouteId }) => {
       return acc;
     }, {});
 
+    // Keep the default route (used in Nop / Std ensemblers), if any, on top
```

Comment:
The previous blue cell highlight for the default route (shown below) is now removed, but I left the default route to continue to be on sorted on the top.

![Screenshot 2022-05-17 at 2 32 55 PM](https://user-images.githubusercontent.com/23465343/168744413-1fdaa19b-52a3-4083-a9d3-2755810547cd.png)